collectors/30/kafka_offset_collector.py

```diff
@@ -40,60 +45,78 @@ def format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offse
 # KAFKA OFFSET MONITORING
 
 
-def get_partitions(zk, group, topic):
-    return zk.get_children(KafkaPaths.topic_offsets(group, topic))
+def get_partitions(zk, topic):
+    path = KafkaPaths.partitions(topic)
+    return map(int, zk.get_children(path))
+
+
+def get_kafka_topics(zk):
+    path = KafkaPaths.topics()
+    return [str(topic) for topic in zk.get_children(path) if topic not in BLACKLIST]
```

Comment:
test topics should be rare - there should only have been this one because i added it in the beginning and kafka is bad at deleting topics (or i'm bad at figuring out how to do this). i don't want us to generally have test topics here, for instance i don't think this is where we should put staging or temporary prototype topics, but even if we did, this i don't think is the right place to be working that out